nexus-api/src/routes/v0/bootstrap/user.rs

```diff
@@ -0,0 +1,40 @@
+use crate::Result;
+use crate::{routes::v0::endpoints::USER_BOOTSTRAP_ROUTE, Error};
+use axum::extract::Path;
+use axum::Json;
+use nexus_common::models::bootstrap::{Bootstrap, ViewType};
+use tracing::info;
+use utoipa::OpenApi;
+
+#[utoipa::path(
+    get,
+    path = USER_BOOTSTRAP_ROUTE,
+    description = "Initial payload for all data required to bootstrap the pubky.app application. The client app will request it while the user is performing sign-in in order to pre-populate the client DB",
+    tag = "User",
+    params(
+        ("user_id" = String, Path, description = "User Pubky ID")
+    ),
+    responses(
+        (status = 200, description = "Initial payload to bootstrap the client", body = Bootstrap),
+        (status = 404, description = "User payload not found"),
```

Comment:
```suggestion
        (status = 404, description = "User id requested for bootstrap payload not found"),
```
Does this ever happen? Do we ever return a `Error::UserNotFound { user_id }` if the user does not exist? Or do we mean this error is `Bootstrap payload not found` ?